src/keyword.rs

```diff
@@ -0,0 +1,53 @@
+// Copyright 2016 Mozilla
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+// this file except in compliance with the License. You may obtain a copy of the
+// License at http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software distributed
+// under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+// CONDITIONS OF ANY KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations under the License.
+
+/// Just like Clojure's Keyword. We'll need this as part of EDN parsing,
```

Comment:
Yeah, pushing this into `edn` would make sense. I wasn't too fussed about the location — just getting it down on paper.